code/modules/mob/living/carbon/life.dm

```diff
@@ -56,8 +56,13 @@
 	//Suffocate
 	if(losebreath > 0)
 		losebreath--
+		if(losebreath >= 3) //can't talk from lack of air
+			silent = max(silent, 2)
```

Comment:
this is also going to have a lot of wide-range balance effects you may not have forseen